swap_meet/item.py

```diff
@@ -1,2 +1,32 @@
+import uuid
+
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id=None, condition=0):
+        if id is None:
+            self.id = uuid.uuid4().int
+        else: 
+            self.id = id
```

Comment:
This particular conditional works well! As you get more comfortable with Python, short and concise conditionals like this can be shortened into a ternary operator that will assign a value to an attribute dependent on the outcome of some condition:

```suggestion
        self.id = id if id is not None else uuid.uuid4().int
```